Protocol.md

```diff
@@ -66,6 +66,7 @@ the funding transaction and CETs.
 
 1. type: 42778 (`offer_dlc_v0`)
 2. data:
+   * [`protocol_version`: `bigsize`]
```

Comment:
nit: I think this is backwards, should be ```[`bigsize`:`protocol_version`]```

Reply:
Thanks fixed